### ISSUE TITLE
Page do not update when clicking on a trail on home page

### ISSUE THREAD

**Issue by ai-ri-kr** [Flomp/wanderer#27]
When you're on the home page and click on a trail, the URL in the address bar updates but not the page. The page just scrolls up to the top. To go to the trail you clicked on, you need to update the page.

**Comment by Flomp** [comment]
I currently cannot replicate the problem in any browser. Are there any errors in the dev console?

**Comment by ai-ri-kr** [comment]
Yes, I have disabled WebGL in Firefox and it is because of that.

**Comment by Flomp** [comment]
And enabling WebGL fixes the problem?

**Comment by ai-ri-kr** [comment]
Yepp :) Can you remove the WebGL requirements?

**Comment by Flomp** [comment]
![wanderer_webgl_disabled](https://github.com/Flomp/wanderer/assets/26000991/e8e3838d-dfd2-485d-b0d1-b84c3756b5d4)
I have disabled WebGL by setting `webgl.disabled=true` in my firefox about:config. 
I can now see the error, which is expected because the 3D Model cannot load. However, this does not prevent me from navigating to a route page as you can see. Am I missing something?
